include/stringzilla/stringzilla.h

```diff
@@ -181,6 +175,9 @@
 #endif
 #endif
 
+#include <stdio.h>
```

Comment:
The goal of this header is to be entirely independent from LibC, so we have to remove those includes.